packages/maskbook/src/plugins/Trader/apis/1inch/index.ts

```diff
@@ -0,0 +1,36 @@
+import BigNumber from 'bignumber.js'
+import { first } from 'lodash-unified'
+import { BIPS_BASE, ONE_INCH_BASE_URL } from '../../constants'
+import type {
+    SwapQuoteOneResponse,
+    SwapQuoteOneRequest,
+    SwapOneServerErrorResponse,
+    SwapOneValidationErrorResponse,
+    SwapOneErrorResponse,
+} from '../../types'
+import type { NetworkType } from '@masknet/web3-shared-evm'
+import urlcat from 'urlcat'
+
+export async function swapOneQuote(request: SwapQuoteOneRequest, networkType: NetworkType) {
+    const params: Record<string, string> = {}
+    Object.entries(request).map(([key, value]) => {
+        if (typeof value === 'string') params[key] = value
+    })
+    if (request.slippage) params.slippage = new BigNumber(request.slippage).dividedBy(BIPS_BASE).toFixed()
+
+    const response =
+        (await fetch(urlcat(ONE_INCH_BASE_URL[networkType], 'swap', params))) ||
+        (await fetch(urlcat(ONE_INCH_BASE_URL[networkType], 'quote', params)))
```

Comment:
```suggestion
    const swapUrl = urlcat(ONE_INCH_BASE_URL[networkType], 'swap', params)
    const quoteUrl = urlcat(ONE_INCH_BASE_URL[networkType], 'quote', params)
    const response = (await fetch(swapUrl)) ||  (await fetch(quote))
```

Better to write code easy to understand within a glance.
We have so many dependency libraries, saving 1 or 2 lines of code doesn't really matter. Clean code would be better.